### PR TITLE
Create Media Playback issue template

### PR DESCRIPTION
**Pull request by EraYaN**
**Changes**
Added media playback issue template. We might want to add some more information requests in there.

### CODE REVIEW

**Comment by PrplHaz4** [comment]
What is the "Media Info" tool?  Also, is it helpful to try and capture hw accel info here too (if ffmpeg output isn't enough)?
- GPU
- Jellyfin HW Acceleration (NVENC, VAAPI...etc)

**Comment by EraYaN** [comment]
MediaInfo: https://mediaarea.net/en/MediaInfo

Very useful.

File: .github/ISSUE_TEMPLATE/media_playback.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+name: Media playback issue
+about: Create a media playback issue report
+title: ''
+labels: mediaplayback
+assignees: ''
+
+---
+
+**Media Info of the file**
+<!-- Use the Media Info tool (set to text format) or copy the info from the web ui for the file with the playback issue. -->
```
Comment (cvium) [inline .github/ISSUE_TEMPLATE/media_playback.md]: Maybe include a link? https://mediaarea.net/en/MediaInfo

File: .github/ISSUE_TEMPLATE/media_playback.md [hunk 2]
```diff
@@ -17,9 +17,11 @@ assignees: ''
 <!-- If available, add screenshots of the stats for nerds screen to help show the issue problem. -->
 
 **Server System (please complete the following information):**
- - OS: [e.g. Docker, Debian, Windows]
+ - OS: [e.g. Docker on Linux, Docker on Windows, Debian, Windows]
  - Jellyfin Version: [e.g. 10.0.1]
+ - Hardware settings & device: [e.g. NVENC on GTX1060, VAAPI on Intel i7 87000K]
```
Comment (Bond-009) [inline .github/ISSUE_TEMPLATE/media_playback.md]: One to many zeroes 8700k

Comment (EraYaN) [reply .github/ISSUE_TEMPLATE/media_playback.md]: Right, fixed.

**Comment by EraYaN** [comment]
I will squash all these commits before we merge this.

**Comment by joshuaboniface** [comment]
I squash-merged it @EraYaN

**Pull request merged**